### ISSUE TITLE
Test and document alternative solutions to increasing the font weight

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-design-system#3964]
## What
Test and document alternative solutions to increasing the font weight. 

Increasing the font weight of the navigation items will:

- make it harder for users using high contrast colour modes to distinguish a navigation item from any other text in the navigation bar, for example the service name.
- remove any visual hierarchy between other content in the 'before page content' area.

## Why
Users have let us know they find it hard to read, it is also recognised as failing with a new method of measuring colour contrast of text on a background.

## Additional context
Issues on alphagov repositories
- [Links on grey have low colour contrast #3862](https://github.com/alphagov/govuk_publishing_components/issues/3862)
- [Problems with New Transport font for vision impaired users #2755](https://github.com/alphagov/govuk-frontend/issues/2755#top)
Preliminary work into new APCA method
- [Colour styles tested for contrast via the Bridge-PCA method](https://docs.google.com/spreadsheets/d/17-LEU0x5UyZbXJ66d0dR6-SQXTuPosFlLN8-3qygw-E/edit?gid=766898507#gid=766898507) showing other components affected by this colour combination

## Done when
- [x] The contrast of tints of the grey background have been tested with the new colour contrast method
- [x] The contrast of tints of the blue text colour have been tested with the new colour contrast method

**Comment by CharlotteDowns** [comment]
| Name                                     | Background colour | Text colour | APCA Contrast score | Minimum font size for 400 font weight (GDS Transport light) | Minimum font size for 700 font weight (GDS Transport bold) | Pass on mobile | Pass on desktop |
| ---------------------------------------- | ----------------- | ----------- | ------------------- | ----------------------------------------------------------- | ---------------------------------------------------------- | -------------- | --------------- |
| Navigation item (on white)               | #ffffff           | #1d70b8     | 75.1                | 18px                                                        | 14px                                                       | Y              | Y               |
| Navigation item (50% lighter background) | #f9f9f8           | #1d70b8     | 71.5                | 19.5px                                                      | 14.5px                                                     | N              | N               |
| Navigation item (20% lighter background) | #f5f5f4           | #1d70b8     | 69.1                | 20.25px                                                     | 15px                                                       | N              | N               |
| Navigation item (as is)                  | #f3f2f1           | #1d70b8     | 67.5                | 20.75px                                                     | 15px                                                       | N              | N               |

**Comment by CharlotteDowns** [comment]
## Navigation item (on white)
### Pros
- doesn't look unusable
### Cons
- doesn't stand out as much (which becomes more important with other elements like the 'switcher')

## Navigation item (50% lighter background)
### Pros
- Less WCAG risks (apart from the obvious contrast)
### Cons
- perception as white, advantages of it being a different colour to distinguish are lost

## Navigation item (20% lighter background)
### Pros
- Less WCAG risks (apart from the obvious contrast)
### Cons
- Not 25% as styles guidance suggests
- perception as white, advantages of it being a different colour to distinguish are lost

## Navigation item (with 10% darker text) 
Could also be done with `filter: brightness(.9);` combined hex code would be: `#1B64A6` (from picker, rough)
### Pros
- precedent of doing something similar before with text colours (notification banner, existing header)
- similar enough that users may not notice
### Cons
- not using a colour that's in our existing colour palette is not best practice
